src/xenia/vfs/devices/disc_image_device.cc

```diff
@@ -127,9 +127,11 @@ bool DiscImageDevice::ReadEntry(ParseState* state, const uint8_t* buffer,
   entry->attributes_ = attributes | kFileAttributeReadOnly;
   entry->size_ = length;
   entry->allocation_size_ = xe::round_up(length, bytes_per_sector());
-  entry->create_timestamp_ = 0;
-  entry->access_timestamp_ = 0;
-  entry->write_timestamp_ = 0;
+
+  // Set to January 1, 1601 (UTC) in 100-nanosecond intervals
```

Comment:
Yes it should - you're using the magic number to set the time to Jan 1, 1970 from the timebase of Jan 1, 1601.